dataline-integrations/singer/bigquery/destination/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7-alpine
+
+WORKDIR /singer
+
+ENV VIRTUAL_ENV=/singer/env
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+RUN apk add --no-cache gcc musl-dev libffi-dev
```

Comment:
do you actually need that?

Reply:
yeah all three are necessary for the pip install to complete